packages/cdk/lib/constants.ts

```diff
@@ -100,8 +100,8 @@ runcmd:
 # Setup ecs additions
```

Comment:
The code above also needs to be && instead of pipe
 command -v aws || sleep 5 |

Reply:
Good catch. Changed it as well.